application/package.json

```diff
@@ -56,5 +56,11 @@
     "@types/react-dom": "$@types/react-dom",
     "react": "$react",
     "react-dom": "$react-dom"
-  }
+  },
+  "postcss": {
+    "plugins": {
+      "tailwindcss": {}
+    }
+  },
+  "browserslist": ["last 1 version", "> 1%", "not dead"]
```

Comment:
build only reads from the project - so these configurations are not used

Reply:
See previous comment.